assets/src/edit-story/components/canvas/useCanvasKeys.js

```diff
@@ -135,17 +138,18 @@ function useCanvasKeys(ref) {
   useGlobalKeyDownEffect(
     { key: ['up', 'down', 'left', 'right'], shift: true },
     ({ key, shiftKey }) => {
-      const dirX = getArrowDir(key, 'ArrowRight', 'ArrowLeft');
-      const dirY = getArrowDir(key, 'ArrowDown', 'ArrowUp');
-      const delta = shiftKey ? 1 : MOVE_COARSE_STEP;
+      if (isEditing) {
+        return;
+      }
```

Comment:
@miina, I've added this to address that under some circumstances, you could still move the whole element around rather than the masked image. I think it makes sense.

This also impacts text editing, where you could select some text in edit mode, click "nothing" in the design panel, and then use arrow keys to move the whole text field around. It was actually pretty weird, so now both of those are disabled.

I think that's a fair result.